apps/betterangels/src/app/(private-screens)/add-note/Purpose/index.tsx

```diff
@@ -114,7 +114,7 @@ export default function Title(props: IPurposeProps) {
           error={!!errors.purpose}
```

Comment:
Hey @mikefeldberg 

I gave you an approval, but I noticed something not related to the subject. 
When I go to either Provided or Requested Services, I didn't see the Search box. Can you put it back or does this PR just need to get merged and it will have the search box back?

<img src="https://github.com/user-attachments/assets/d1314b1d-6f47-4b22-b4f1-cc4c7372de5e" width="360"  />
<img src="https://github.com/user-attachments/assets/44066910-524e-4bd4-9591-80bff70ab493" width="360"  />

Reply:
I'm not sure what's going on with service search but it's an unrelated issue